File: cdb-web/src/app/models/retornoCalculo.ts
import { DecimalPipe } from "@angular/common";
import { Imposto } from "./imposto";

export class RetornoCalculo {

  valorInicial!: DecimalPipe;
  valorBruto!: DecimalPipe;
  imposto!: Imposto;
  prazo!: number;
  valorLiquido!: DecimalPipe;

}

File: cdb-back/CDB.Domain/Entities/RetornoCalculo.cs
namespace CDB.Domain.Entities
{
    public class RetornoCalculo
    {

        public decimal ValorInicial { get; set; }
        public int Prazo { get; set; }
        public decimal ValorBruto { get; set; }
        public decimal ValorLiquido { get; set; }
        public Imposto? Imposto { get; set; }

    }
}
